magclassic/__init__.py

```diff
@@ -1,5 +1,6 @@
 from uber.common import *
 from magclassic._version import __version__
+from magclassic.automated_emails import *
```

Comment:
In this particular case, it works to put this import statement here, because nothing in ``magclassic/automated_emails.py`` relies on the ``magclassic`` plugin config (which is completely empty at the moment if memory serves, since we haven't found a need for any config specific to that event).

However, in general we'll need to put this kind of importing after the config is parsed.